### PR TITLE
Add opkg as package manager

### PR DESCRIPTION
**Pull request by autinerd**
opkg is a lightweight package manager, from the syntax similar to apt, often used in embedded linux systems.

It also (maybe) adds a falsehood: You can search for a package in a package manager.
opkg has no search command, you have to list all packages and then perform a grep or something like that.

### CODE REVIEW

File: meta_package_manager/managers/opkg.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+from ..base import PackageManager
+from ..platform import LINUX
+
+
+class OPKG(PackageManager):
+
+    platforms = frozenset([LINUX])
+
+    requirement = '>= 1.0.0'
```
Comment (kdeldycke) [inline meta_package_manager/managers/opkg.py]: Latest `opkg` version seems to be 0.4.1: http://git.yoctoproject.org/cgit/cgit.cgi/opkg/commit/?id=fdf77e6674bad59e60d0cb58812cf1669676950e

Can you at least change the required version to that that one?

**Comment by kdeldycke** [comment]
That a nice niche package manager I did not know about. I see no problem to merge it upstream. But can you please add a couple of additional changes in this MR, like:
* Updating the changelog, as in: https://github.com/kdeldycke/meta-package-manager/commit/e46a296f717c12edb9dfab12543934e13a2c9693
* Updating the readme, to add `opkg` in the table: https://github.com/kdeldycke/meta-package-manager/commit/1be2bdb498313369e3bf91b238234e91fca036e6

File: meta_package_manager/managers/opkg.py (hunk 1 above)
Comment (autinerd) [reply meta_package_manager/managers/opkg.py]: Oh, I overlooked this line, sorry. I corrected it.

**Comment by autinerd** [comment]
I have added the changes. When something is still missing, please let me know!

**Comment by kdeldycke** [comment]
Tests are failing because of coding style but that's fine to me. I'll fix that later. Thanks for the contribution! :)

**Pull request merged**